From 2ff00646912b06c45ce45b9b525ae09c3500fcc0 Mon Sep 17 00:00:00 2001
From: Jon Craig <jcraig@opencsw.org>
Date: Sat, 11 Jan 2014 21:38:04 +0100
Subject: [PATCH] Update macros used in Makefile.am for modern convention

INCLUDES is a deprecated name in Makefile.am.  Use AM_CPPFLAGS instead.
---
 Makefile.am         | 2 +-
 plugins/Makefile.am | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index e4b0fc1..a8709f2 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -26,7 +26,7 @@ PROGRAM  = $(PKGNAME)-$(VERSION)
 DEFS      = @DEFS@
 INCS      = @INCS@
 
-INCLUDES  = -I. $(INCS) ${RRD_INC}
+AM_CPPFLAGS  = -I. $(INCS) ${RRD_INC}
 LIBS      = 
 BASE_LIBS = @LIBS@ @CORELIBS@  ${RRD_LIB}
 AS        = @AS@
diff --git a/plugins/Makefile.am b/plugins/Makefile.am
index 88e193c..3c412e7 100644
--- a/plugins/Makefile.am
+++ b/plugins/Makefile.am
@@ -37,7 +37,7 @@ SUFFIXES    = .so
 # 
 plugindir = $(libdir)/ntop/plugins
 
-INCLUDES = -I.. @INCS@ ${RRD_INC}
+AM_CPPFLAGS = -I.. @INCS@ ${RRD_INC}
 
 #
 # The meat for ntop
-- 
1.8.4.1

